avm/src/daemon/lrh.rs

```diff
@@ -46,7 +46,7 @@ impl LogRendezvousHash {
 
   // Runs a binary search for the record whose hash is closest to the key hash without
   // going over. If none are found, the *last* record in the list is returned as it wraps around.
-  pub fn get_id_for_key(&self, key: &str) -> &str {
+  pub fn _get_id_for_key(&self, key: &str) -> &str {
```

Comment:
do we even need this?

Reply:
it will be used later so suppressing the warning for now